### ISSUE TITLE
Add support for other iOS versions and more devices

### ISSUE THREAD

**Issue by V3rochka** [alephsecurity/xnu-qemu-arm64#1]
### Description
Every other iOS version or device will bring other challenges to cope with. The next version we would like to support is iOS 13.

**Comment by aBradleyUno** [comment]
Are you guys jumping straight from QEMU to XNU? (Sorry, still reading the posts)

**Comment by jonyafek** [comment]
Yes, you can say that. Using QEMU emulation to execute XNU. In the context of this issue, currently working on porting this to iPhone 11 - iOS 14.

**Comment by TylerJaacks** [comment]
What sort of work would need to be added to get new versions to work? What would happen if I tried the steps on a modern iPhone and iOS? Where would things go wrong?

**Comment by jonyafek** [comment]
Yes, the device tree is different and the boot process is different and there are some patches that we do on boot. I started working on it and it is about halfway done but I can't find the time to complete the task. I hope to get to it soon.
